### PR TITLE
Add CORS headers directly instead of in the proxy

### PR DESCRIPTION
**Pull request by kelunik**

### CODE REVIEW

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java [hunk 1]
```diff
@@ -90,7 +94,7 @@ else if (config.database.maintainInterval < 0)
         }
     }
 
-    private static void boot(DatabaseManager databaseManager, List<Platform> platforms, Credentials readOnly, int cleanupInterval) throws SQLException {
+    private static void boot(DatabaseManager databaseManager, List<Platform> platforms, Credentials readOnly, int cleanupInterval, String origin) throws SQLException {
```
Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:97]: Just passing the `config` here? @marcelhollerbach @LeanderK

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/config/Deployment.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package edu.kit.ipd.crowdcontrol.objectservice.config;
+
+public class Deployment {
+    public String origin;
+}
```
Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/config/Deployment.java]: can you add comments like in the other config classes?

File: src/main/resources/.gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 config.properties
+config.yml
```
Comment (marcelhollerbach) [inline src/main/resources/.gitignore:2]: why do you have a nested .gitignore?

File: src/main/resources/config.yml [hunk 4]
```diff
@@ -6,12 +5,15 @@ database:
     dialect: MYSQL
     #the writing user
     writing:
-        user: root
-        password:
+        user: crowdcontrol
+        password: hH2CAd6nttG2B7qc
```
Comment (marcelhollerbach) [inline src/main/resources/config.yml]: trololol

File: src/main/resources/config.yml [hunk 5]
```diff
@@ -1,4 +1,3 @@
-
 database:
```
Comment (marcelhollerbach) [inline src/main/resources/config.yml]: did I push this file ?? Fuuuuck

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java (hunk 1 above)
Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:97]: Yeah fine with that. I dont get the purpose of the boot method at all...

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/config/Deployment.java (hunk 2 above)
Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/config/Deployment.java]: done.

File: src/main/resources/config.yml [hunk 6]
```diff
@@ -1,33 +0,0 @@
-
```
Comment (marcelhollerbach) [inline src/main/resources/config.yml:1]: thx.

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/config/Deployment.java [hunk 7]
```diff
@@ -0,0 +1,8 @@
+package edu.kit.ipd.crowdcontrol.objectservice.config;
+
```
Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/config/Deployment.java:2]: please also for the datastruct - who knows what will be added here ... ports etc. maybe.

File: src/main/resources/.gitignore (hunk 3 above)
Comment (kelunik) [inline src/main/resources/.gitignore:2]: Usually do that for config directories to have everything about configs at one place and not in the rather long main `.gitignore`.

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/config/Deployment.java (hunk 7 above)
Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/config/Deployment.java:2]: done.

**Pull request merged**